types/trusted-types/test/browser.ts

```diff
@@ -104,11 +104,11 @@ trustedHTML = trustedScript;
 // @ts-expect-error
 new TrustedHTML();
 
-// $ExpectType typeof TrustedHTML
+// $ExpectError
 TrustedHTML;
-// $ExpectType typeof TrustedScript
+// $ExpectError
 TrustedScript;
-// $ExpectType typeof TrustedScriptURL
+// $ExpectError
```

Comment:
These are not meant to be used as values e.g. for `new TrustedHTML()`.

Reply:
FYI these should be `@ts-expect-error`; the `$ExpectError` form is going away and has been dropped from everything in the repo (see lines above).

Reply:
ah sweet. will change

Reply:
Using `@ts-expect-error` now